modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/ProductResourceTest.java

```diff
@@ -114,6 +137,30 @@ protected Long testGraphQLGetChannelProduct_getChannelId()
 		return _commerceChannel.getCommerceChannelId();
 	}
 
+	@Override
+	protected Product testGraphQLGetChannelProductByFriendlyUrlPath_addProduct()
+		throws Exception {
+
+		return _addCPDefinition(randomProduct());
+	}
+
+	@Override
+	protected Long testGraphQLGetChannelProductByFriendlyUrlPath_getChannelId()
+		throws Exception {
+
+		return _commerceChannel.getCommerceChannelId();
+	}
+
+	@Override
+	protected String
+			testGraphQLGetChannelProductByFriendlyUrlPath_getFriendlyUrlPath()
+		throws Exception {
+
+		return _addCPDefinition(
+			randomProduct()
+		).getName();
+	}
```

Comment:
I don't think it's correct because this method should return the friendlyUrlPath of a product